src/main/java/uk/gov/pay/adminusers/model/CompleteInviteResponse.java

```diff
@@ -14,8 +15,9 @@ public class CompleteInviteResponse {
     @Schema(example = "89wi6il2364328")
     private String serviceExternalId;
 
-    public CompleteInviteResponse(@JsonProperty("invite") Invite invite) {
+    public CompleteInviteResponse(Invite invite, String userExternalId) {
```

Comment:
Good point, could be final.

The `JsonProperty` annotation was removed because it was superfluous. Jackson will serialise using the field name which is already `invite` and `@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)` makes the other fields serialise as snake case. And I'm not even sure what it does if it's used in the constructor for a response class 🤔 .

We only need to include `JsonProperty` when we want the field to be serialised with a different name to the default.